rgbd_msgs/package.xml

```diff
@@ -1,20 +1,26 @@
 <?xml version="1.0"?>
-<package>
+<?xml-model
+  href="http://download.ros.org/schema/package_format3.xsd"
+  schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>rgbd_msgs</name>
-  <version>0.0.0</version>
+  <version>0.1.0</version>
   <description>The rgbd_msgs package</description>
 
-  <maintainer email="amigo@todo.nl">TODO</maintainer>
+  <maintainer email="MatthijsBurgh@outlook.com">Matthijs van der Burgh</maintainer>
 
   <license>TODO</license>
 
   <buildtool_depend>catkin</buildtool_depend>
+
```

Comment:
I think we have more dependencies, e.g., `tue_serialization`. At least check `CMakeLists.txt`?

Reply:
I don't know how I missed that one

Reply:
This is `rgbd_msgs`, so not needed here.